async-openai/src/vector_store_files.rs

```diff
@@ -75,3 +75,58 @@ impl<'c, C: Config> VectorStoreFiles<'c, C> {
             .await
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::Client;
+    use crate::types::{CreateFileRequest, CreateVectorStoreFileRequest, CreateVectorStoreRequest, FileInput, FilePurpose};
+
+    #[tokio::test]
+    async fn vector_store_file_creation_and_deletion() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+        let client = Client::new();
+
+        // Create a file
+        let file_handle = client
+            .files()
+            .create( CreateFileRequest {
+                file: FileInput::from_vec_u8(
+                    String::from("meow.txt"),
+                    String::from(":3").into_bytes()
+                ),
+                purpose: FilePurpose::Assistants
+            }).await?;
+
+        // Create a vector store
+        let vector_store_handle = client
+            .vector_stores()
+            .create( CreateVectorStoreRequest {
+                file_ids: None,
```

Comment:
You can actually attach `file_handle` in the request without a name too and it will work. That way name is still optional 

https://github.com/64bit/async-openai/issues/229#issuecomment-2155615304

Reply:
Modified in [following commit](https://github.com/64bit/async-openai/pull/230/commits/29a790a10774f017e9c46cef32817f66e2191fe0)